models/selectCommentsByArticleId.models.js

```diff
@@ -0,0 +1,41 @@
+const db = require("../db/connection");
+
+exports.selectCommentsByArticleId = (
+  article_id,
+  sort_by = "created_at",
+  order = "desc"
+) => {
+  const validSortBy = ["created_at"];
+  const validOrderBy = ["asc", "desc", "ASC", "DESC"];
+
+  if (sort_by && !validSortBy.includes(sort_by)) {
+    return Promise.reject({ status: 400, msg: "Invalid Sort Request" });
+  }
+
+  if (sort_by && !validOrderBy.includes(order)) {
+    return Promise.reject({ status: 400, msg: "Invalid Order Request" });
```

Comment:
As advised above, all of this logic doesn't need to be here yet. You have correctly put the sort_by and order in the parameters as defaults, so the rest of this logic can be removed until we need it later! 🙂